runtime/activations/activations.go

```diff
@@ -115,6 +123,14 @@ func (a *ValueTypeActivation) Set(name string, value ValueType) {
 //
 type ValueTypeActivations struct {
 	activations []*ValueTypeActivation
+	memoryGauge common.MemoryGauge
+}
+
+func NewValueTypeActivations(memoryGauge common.MemoryGauge) *ValueTypeActivations {
+	// No need to meter.
+	return &ValueTypeActivations{
```

Comment:
This does not seem to be used? 

In case it is: Why do we not need to meter it?

Reply:
This is the template code used for code-generation.

In the variable activation case, 'activation list' is created only once per interpreter. So we don't need to meter it.

I didn't want to add the word 'interpreter' to the template, and wanted to keep the comment also generic. Maybe can say  "created only once per execution" in the comment. Or do you have any suggestions?

Reply:
Oh, I didn't realize this was the templated code, thank you for clarifying 👍 

Maybe this function should be just called `NewUnmetered...` and not take a memory gauge, given that it does not meter?

Reply:
Even though this constructor doesn't meter `Activations` , it keeps a ref to the memory gauge, and uses it to meter each new `Activation` and activation-entries initialization. This was done to avoid having to pass down the memory-gauge for each method.
i,e: it doesn't meter the outer list, but meter each entry in the list

Reply:
Ah, OK. Could you please add this as a comment to the function?